package.json

```diff
@@ -18,9 +18,10 @@
     "dev": "tsdx watch",
     "build": "tsdx build",
     "prepublishOnly": "npm run build",
-    "pretest": "npm run lint && npm run build",
     "test": "tsdx test --env=jsdom",
```

Comment:
If test is testing the compiled files we need to run build script before, otherwise we are going to test old files.

Reply:
I just made an obvious bug, and without rebuilding, reran the test suite and the tests failed. 

Are there edge cases where this won't always works?

Reply:
Ok, then here we are testing ./src folder and not the compiled code. It is fine, however there could be cases in which ./src works and ./dist doesn't. For example types compilation problems: src works because TS used for tests infers and read correctly the types from the srource code but the compiled .d.ts files are somehow wrong. We can leave it as is, but we must be aware of this edge case. It will surely popup at some point.